src/components/Zodiac/Zodiac.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
```

Comment:
Here we can add import './Zodiac.css'. Add additional styling for an extra 0.5 points :)